.github/workflows/velox_backend_x86.yml

```diff
@@ -1274,3 +1274,21 @@ jobs:
         with:
           name: ${{ github.job }}-test-log
           path: "**/target/*.log"
+
+  build-cudf-centos-9:
+    runs-on: ubuntu-22.04
+    container: apache/gluten:centos-9-jdk8-cudf
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get Ccache
+        uses: actions/cache/restore@v4
+        with:
+          path: '${{ env.CCACHE_DIR }}'
+          key: ccache-centos9-release-shared-${{runner.arch}}-${{github.sha}}
+          restore-keys: |
+            ccache-centos9-release-shared-${{runner.arch}}
+      - name: Build Gluten native libraries
+        run: |
+          df -a
+          bash dev/buildbundle-veloxbe.sh --run_setup_script=OFF --build_arrow=ON --spark_version=3.4 --enable_gpu=ON
+          ccache -s
```

Comment:
we may need to add the cache save step here. it seems this job needs to run 3h with cache miss
https://github.com/apache/incubator-gluten/actions/runs/15553575516/job/43857004930?pr=9933
here's the example code on centos8 tests
https://github.com/apache/incubator-gluten/blob/main/.github/workflows/velox_backend_cache.yml#L82